kolibri/core/analytics/management/commands/profile.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+
+from kolibri.core.analytics.measurements import get_db_info
+from kolibri.core.analytics.measurements import get_kolibri_process_info
+from kolibri.core.analytics.measurements import get_kolibri_use
+from kolibri.core.analytics.measurements import get_machine_info
+from kolibri.core.analytics.measurements import get_process_pid
+from kolibri.core.analytics.measurements import KolibriNotRunning
+from kolibri.core.analytics.pskolibri.common import LINUX
+from kolibri.core.analytics.pskolibri.common import WINDOWS
+
+try:
+    import urllib.request as urlrequest
+except ImportError:
+    import urllib as urlrequest
+
+
+logger = logging.getLogger('profiler')
+
+
+class Command(BaseCommand):
+    """
+    This command will produce a performance.log file with this structure in every line:
+    - Timestamp
+    - Number of Kolibri active sessions
+    - Number of Kolibri active users
+    - Number of Kolibri active users that have interacted in the last minute
+    - Percentage of use of the cpu of the server
+    - Used memory (In Mbytes) in the server
+    - Total memory (In Mbytes) in the server
+    - Number of processes in the server
+    - Percentage of use of cpu by the Kolibri process
+    - Memory (In Kbytes) used by the kolibri process
+    """
+    help = "Logs performance/profiling info in the server running Kolibri"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--num-samples', action='store', dest='num_samples', default=60, type=int,
+            help='Specifies the number of times the profile will take measures before ending'
+        )
+
+    def handle(self, *args, **options):
+        if not LINUX and not WINDOWS:
+            print("This OS is not yet supported")
+            sys.exit(1)
+
+        interval = 10  # the measures are taken every 10 seconds
+        _, port = get_kolibri_process_info()
+        if port:
+            this_pid = get_process_pid()
+            try:
+                urlrequest.urlopen('http://localhost:{port}/api/analytics/activate/{pid}/'.
+                                   format(port=port, pid=this_pid))
```

Comment:
Yes, we can, and I did it in the beginning, but I removed it because @indirectlylit told me to avoid these parameters to make user life easier :)
If you both agree I have no problem in restoring it from the git history.